components/http_foundation/introduction.rst

```diff
@@ -483,6 +483,9 @@ abstracts the hard work behind a simple API::
 
     $response->headers->set('Content-Disposition', $d);
 
+You can add the file's content as usual, for example by using
```

Comment:
Instead of `You can add the file's content as usual...` maybe we should say `Then you must include the file's content as usual...`